packages/common/src/core/schemas/types.ts

```diff
@@ -250,6 +250,13 @@ export interface IUiSchemaMessage {
   labelKey?: string;
   icon?: boolean | string;
   kind?: "brand" | "danger" | "info" | "success" | "warning";
+  link?: {
```

Comment:
I'd suggest leveraging the existing [`HubActionLink`](https://github.com/Esri/hub.js/blob/master/packages/common/src/core/types/ActionLinks.ts) to type this property...or the more specific `IHubExternalActionLink` interface if we want to restrict it to external links. 

It doesn't look like we have an `iconEnd` property on the interface (just an `icon` property), but you can go ahead and add one to `IHubBaseActionLink` if necessary.

There's also no `labelKey` property on the interface, but we shouldn't need one - we should be able to just have a `label` and use the `{{some.key:translate}}` pattern we use elsewhere as long as we're interpolating the translations when we load the `uiSchema`. 

_Aside: my hope in the future is that we'd create a `hub-action-link` component that would accept a `link: HubActionLink` and handle all things around rendering, telemetry, etc. If this were the case, we could have logic that checks if the link is external + has a "blank" target and automatically include the launch icon at the end (as I believe this should always be shown in this case)...this would effectively remove the immediate need for an `iconEnd` in the interface, but I think it's fine to add for now_

Reply:
Done! Great suggestion @juliannemarik!